savemphtxt.m

```diff
@@ -19,18 +18,17 @@ function savemphtxt(node, face, elem, filename)
 n_node = size(node,1);
 n_face = size(face,1);
 n_elem = size(elem,1);
-
-elem(:,1:4)=meshreorient(node(:,1:3),elem(:,1:4));

```

Comment:
I see you deleted this line as well as changing the below if/elseif conditions, are you sure these are equivalent?